README.md

```diff
@@ -93,10 +93,13 @@ maps using Mapnik, many based on this project. Some alternatives are:
 # Maintainers
 
 * Andy Allan [@gravitystorm](https://github.com/gravitystorm/)
-* Matthijs Melissen [@matthijsmelissen](https://github.com/matthijsmelissen/)
 * Paul Norman [@pnorman](https://github.com/pnorman/)
 * Mateusz Konieczny [@matkoniecz](https://github.com/matkoniecz/)
 * Daniel Koć [@kocio-pl](https://github.com/kocio-pl)
 * Christoph Hormann [@imagico](https://github.com/imagico)
 * Lukas Sommer [@sommerluk](https://github.com/sommerluk)
 * Joseph Eisenberg [@jeisenbe](https://github.com/jeisenbe)
+
+## Previous maintainers
```

Comment:
in case of creating this section also https://github.com/gravitystorm/openstreetmap-carto/commit/d3e54b444fdf70ff16a35d91f75781b4c81c3250 should be mentioned